src/OmnichannelChatSDK.ts

```diff
@@ -53,6 +53,7 @@ import { createIC3ClientLogger, createOCSDKLogger, IC3ClientLogger, OCSDKLogger
 import LiveWorkItemDetails from "./core/LiveWorkItemDetails";
 import LiveWorkItemState from "./core/LiveWorkItemState";
 import ConversationMode from "./core/ConversationMode";
+import { defaultMessageTags } from "./core/MessageTags";
```

Comment:
I'd suggest you should add these imports alphabetically. It is much easier to read later